subsystem/metrics_test.go

```diff
@@ -288,6 +307,77 @@ var _ = Describe("Metrics tests", func() {
 		clearDB()
 	})
 
+	FContext("host metrics events", func() {
+		var c *models.Cluster
+		var bootstrap models.Host
+
+		var toProps = func(str string) map[string]interface{} {
+			props := make(map[string]interface{})
+			Expect(json.Unmarshal([]byte(str), &props)).NotTo(HaveOccurred())
+			return props
+		}
+
+		BeforeEach(func() {
+			//start host installation process
+			registerHostsAndSetRoles(clusterID, 3)
+			c = installCluster(clusterID)
+			for _, host := range c.Hosts {
+				waitForHostState(ctx, clusterID, *host.ID, "installing", 10*time.Second)
```

Comment:
Is there a reason you don't use `pollDefaultTimeout`? The jobs may be longer in CI